lib/storext/class_methods.rb

```diff
@@ -23,7 +23,7 @@ def store_attribute(column, attr, type=nil, opts={})
       track_store_attribute(column, attr, type, opts)
       storext_check_attr_validity(attr, type, opts)
       storext_define_accessor(column, attr)
-      store_accessor column, *storext_attrs_for(column)
+      store_accessor(column, attr)
```

Comment:
I guess you can just remove the spec and since this doesn't break anything, this can count as a refactoring. The tests in `spec/storext_spec.rb` are sufficient anyway for the purposes of `storext`.